src/exchange/exchange.app.js

```diff
@@ -17,7 +17,7 @@ angular
             "abstract": true,
             name: "app.microsoft.exchange",
             template: "<div ui-view></div>",
-            translations: ["exchange"]
+            translations: ["exchange", "sharepoint"]
```

Comment:
Sharepoint translations in exchange module ?

Reply:
There are a lot of cross module references like this @marie-j . May be we can take up the clean up as a separate task later on.

Reply:
I did something similar with ovh-module-emailpro who used ecxhange’s translations (https://github.com/ovh-ux/ovh-module-emailpro/pull/70)

Reply:
I searched and found only one occurrence of sharepoint translation : `sharepoint_activate_which_service_label` (https://github.com/ovh-ux/ovh-module-exchange/blob/fix/translation-manager-web/src/exchange/order/order.html#L34) 
So it will be great to import it in `ovh-module-exchange` (conserve `qtlid` and rename-it `exchange_activate_which_service_label`) and remove `sharepoint` translation dependency.